File: app/src/main/java/com/botijasoftware/utils/materials/passoptions/PassOptionMaskDepth.java
package com.botijasoftware.utils.materials.passoptions;

import android.opengl.GLES10;

import com.botijasoftware.utils.materials.Material;

public class PassOptionMaskDepth extends PassOption {

	@Override
	public void set(Material material) {
		GLES10.glDepthMask(true);
	}

	@Override
	public void unSet() {
		GLES10.glDepthMask(false);
	}

}

File: app/src/main/java/com/botijasoftware/utils/materials/passoptions/PassOptionMaskDepth.kt
package com.botijasoftware.utils.materials.passoptions

import android.opengl.GLES10

import com.botijasoftware.utils.materials.Material

class PassOptionMaskDepth : PassOption() {

    override fun set(material: Material) {
        GLES10.glDepthMask(true)
    }

    override fun unSet() {
        GLES10.glDepthMask(false)
    }

}